binary_transparency/firmware/api/http.go

```diff
@@ -19,6 +19,8 @@ import "fmt"
 const (
 	// HTTPAddFirmware is the path of the URL to publish a firmware entry.
 	HTTPAddFirmware = "ft/v0/add-firmware"
+	// HTTPAddAnnotationMalware is the path of the URL to publish annotations about malware scans.
+	HTTPAddAnnotationMalware = "ft/v0/add-annotation-malware"
```

Comment:
Worth discussing this vs. more general `add-annotation`?

Reply:
Yeah, this is one of the discussion points of the strawman. I think there are arguments in all kinds of directions. It ties in with whatever we choose to do with the annotation type enum too.

Reply:
Options:
 1. Generic `AddStatement` that takes a `SignedStatement`
 2. `AddFirmware` and `AddAnnotation`
 3. `AddFirmware`, `AddAnnotationMalware`, `AddAnnotationX`, ...

My preference is probably [3, 1, 2].

Reply:
We'll go with 3 for now.